### ISSUE TITLE
Not receiving recovery email

### ISSUE THREAD

**Issue by kwadner** [keybase/keybase-issues#3730]
I'm attempting to do a reset and recovery of my account since I've not used it for.. a lot time. However, I'm not receiving the recovery email in my inbox or spam. Requests were initiated yesterday and today. Keybase username name "k i e l w a d n e r" without spaces. Email is same as GitHub. Suggestions on how to proceed?

**Comment by heronhaye** [comment]
Hi, what specifically did you try to do? Recover username, password, or start account reset?
I'm seeing an email for recover username that went out around a day ago but no reset process started.

**Comment by kwadner** [comment]
Hi @heronhaye, I'm attempting an account reset but you are correct I did do a username recovery. 

My steps:
1. Account Login
2. Click on "Forgot Password?"
3. Enter the username above and press continue
4. On the warning screen about account recovery press "Start account reset"
5. Click close on message box about checking email.

The steps above were performed again around 21:18 UTC. 
The _first_ time I tried, the UI briefly showed a red message about a database error, but I don't recall which step and haven't seen it since.

**Comment by heronhaye** [comment]
Got it. Please try again tomorrow.

**Comment by kwadner** [comment]
Received the following error when trying to reset account. I continued on but did not receive an email.
![DB Error](https://user-images.githubusercontent.com/2546881/70261058-7f5a5f00-1746-11ea-835e-9eb472d33aba.png)

**Comment by kwadner** [comment]
@heronhaye any suggestions for this? I am still receiving the DB error shown above when trying to reset.

**Comment by duncanbeevers** [comment]
I had attempted this process a few days ago, and saw the same `DB error` as [mentioned previously](https://github.com/keybase/keybase-issues/issues/3730#issuecomment-562256507).

Today I attempted the process again and received a success message, but no recovery email seems to have been sent to the address associated with the account.

**Comment by heronhaye** [comment]
@duncanbeevers Are you doing this from the website? Are you logged in and going through the flow at https://keybase.io/account/reset_me, or are you pressing "Forgot password" in the login modal? Or are you resetting from the app?

**Comment by duncanbeevers** [comment]
@heronhaye This is a logged-out, account recovery/hard reset flow.
![screenshot](https://user-images.githubusercontent.com/7367/71637580-2a055680-2bfb-11ea-8d2f-70e485bc92c3.png)

**Comment by kwadner** [comment]
@heronhaye  I am seeing that same "Autoreset is not available on accounts without devices" message at the top. It is from an iOS device with a fresh install of the app after entering my username and clicking "Forgot password".

When I pressed Start Account reset I see a message similar to @duncanbeevers did about checking my email. No message was received, including in the spam folder. I've pressed "Send it again" and do not receive an email.

**Comment by heronhaye** [comment]
Thanks for the reports. We're going to fix this bug soon and let you all know.

**Comment by heronhaye** [comment]
@duncanbeevers Just to be clear, you've both lost all your devices and forgotten your password?

**Comment by duncanbeevers** [comment]
@heronhaye This is for an account to which I invited a close friend. They signed up, but seem never to have added devices.

**Comment by duncanbeevers** [comment]
@heronhaye Feel free to message me on keybase @duncanbeevers should you wish further details.

**Comment by heronhaye** [comment]
@duncanbeevers Ah, you should've said that: I was looking up information for your own user. I will DM you.

**Comment by kwadner** [comment]
> @duncanbeevers Just to be clear, you've both lost all your devices and forgotten your password?

Correct. I signed up years ago but never used it. Might have been all the way back when it was invite only honestly. Not sure what device (or if there was one?) it was tied to. I had a team want to use this now and I didn’t want dual identities for me.

**Comment by heronhaye** [comment]
@duncanbeevers @kwadner Please try entering the reset flow again, thanks!

**Comment by kwadner** [comment]
Looks to be working. I received the email after the reset, was able to confirm reset from the link, and was told about the three day hold.

Thanks.

**Comment by heronhaye** [comment]
Great, let me know if you have any problems completing it in 3 days.

**Comment by sebastianlach** [comment]
Hi, I still have the same problem. When I clicked "Start account reset" I received "Check your email or phone.
We are sending instructions to your email address or phone number." But I haven't received any mail. I doesn't seem like email provided is the issue because when I try to recover username email is delivered. Could you help me with this issue? It's really annoying. Thanks in advance.
